pyproject.toml

```diff
@@ -0,0 +1,131 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "python-libs"
+version = "1.0.0"
+description = "Common XCP-ng Python classes"
+requires-python = ">=2.7"
```

Comment:
Versions older than 3.6 should probably be disallowed here (with the exception of 2.7 for the time being)

Reply:
Done in PR 27!